config.yml

```
@@ -543,10 +543,35 @@ nodes:
     fields:
       - name: key
         type: node
+        comment: |
+          The key of the association. This can be any node.
```

_Almost_ any node. It can't be a void statement, as in an `alias` statement. How about something like:

```suggestion
          The key of the association. This can be any node that represents a non-void expression.
```